interpreter/cling/lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -420,7 +421,10 @@ namespace {
     R.clear();
     R.setLookupName(&m_Context->Idents.get("copyArray"));
     m_Sema->LookupQualifiedName(R, NSD);
-    assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
+    /* If I am not being called from a child Interpreter, only then do
```

Comment:
Perfect! (But instead of "this check can go away" I'd rephrase as "this assert can be re-activated")